### PR TITLE
GitHub Subscribe Org - Second PR Post Merge

### PR DESCRIPTION
**Pull request by benschuster788**
This includes the fixes for pagination of Github ListOrgs API call to retrieve all of an organizations repos as the response is limited to 30 repos.  Added in logging if p.AddSubscription fails.

### CODE REVIEW

File: server/subscriptions.go [hunk 1]
```diff
@@ -108,13 +108,26 @@ func (p *Plugin) SubscribeOrg(ctx context.Context, githubClient *github.Client,
 
 	listOrgOptions := github.RepositoryListByOrgOptions{
 		Type: "all",
+		ListOptions: github.ListOptions{
+			Page: 1,
```
Comment (hanzei) [inline server/subscriptions.go]: The first page is 0 if I recall correctly.

File: server/subscriptions.go [hunk 2]
```diff
@@ -108,13 +108,26 @@ func (p *Plugin) SubscribeOrg(ctx context.Context, githubClient *github.Client,
...
+		},
 	}
-	repos, _, err := githubClient.Repositories.ListByOrg(ctx, org, &listOrgOptions)
-	if repos == nil || err != nil {
-		if err != nil {
-			mlog.Error(err.Error())
+	repos := []*github.Repository{}
+	fetchingRepos := true
+
+	for fetchingRepos {
```
Comment (hanzei) [inline server/subscriptions.go]: There is a idiomatic way to write this loop. Would you please take a look at https://github.com/google/go-github#pagination? I'm 3/5 that you should just follow the example there.

File: server/subscriptions.go [hunk 3]
```diff
@@ -126,6 +139,7 @@ func (p *Plugin) SubscribeOrg(ctx context.Context, githubClient *github.Client,
 		}
 
 		if err := p.AddSubscription(fmt.Sprintf("%s/%s", org, repo), sub); err != nil {
+			mlog.Error(err.Error())
```
Comment (hanzei) [inline server/subscriptions.go]: :+1:

**Comment by hanzei** [comment]
Thanks for fixing this @benschuster788! :+1:

**Comment by jasonblais** [comment]
Hey @benschuster788 - let me know if you have any questions about above, or if we can help guide you to finishing this contribution. Really appreciate your help here!

**Comment by jasonblais** [comment]
Hey @benschuster788 - I'll close the PR due to inactivity, but leave the branch open if the work is continued later on :)

**Pull request closed without merging**